### PR TITLE
Option for the operator to control a single namespace

### PR DESCRIPTION
**Pull request by andreaTP**
With: https://github.com/lightbend/cloudflow-helm-charts/pull/23

### CODE REVIEW

File: core/cloudflow-operator/src/main/scala/cloudflow/operator/Settings.scala [hunk 1]
```diff
@@ -131,6 +132,11 @@ final case class Settings(config: Config) extends Extension {
   val flinkEnabled = config.getBoolean(s"$root.flink-enabled")
   val sparkEnabled = config.getBoolean(s"$root.spark-enabled")
 
+  lazy val controlledNamespace: Option[String] = Try(sys.env("CONTROLLED_NAMESPACE")) match {
```
Comment (RayRoestenburg) [inline core/cloudflow-operator/src/main/scala/cloudflow/operator/Settings.scala:135]: what about getting it from config and adding a default in the config lie we do in other places?

Comment (andreaTP) [reply core/cloudflow-operator/src/main/scala/cloudflow/operator/Settings.scala:135]: We can, but, in this specific case, I decided to use an env variable to make it easier and clean in the helm chart.
e.g. https://github.com/lightbend/cloudflow-helm-charts/pull/23/files#diff-08bec0d8da17f5482f6840b57e144767f8250cafd31b80a37e13e494d2236b71R5

Having the default config would require some additional logic to be implemented there.

Comment (RayRoestenburg) [reply core/cloudflow-operator/src/main/scala/cloudflow/operator/Settings.scala:135]: ok

Comment (RayRoestenburg) [reply core/cloudflow-operator/src/main/scala/cloudflow/operator/Settings.scala:135]: (the env var could have been in the reference.conf so it is known what is used, but this is also in the code so it does not really matter)

**Review by RayRoestenburg — APPROVED** [review]

File: core/cloudflow-operator/src/main/scala/cloudflow/operator/Settings.scala (hunk 1 above)
Comment (RayRoestenburg) [reply core/cloudflow-operator/src/main/scala/cloudflow/operator/Settings.scala:135]: (I guess I'm thinking of this as, the reference.conf tells me what can be configured, or which env props can be used to change things, but this is a minor detail)

Comment (andreaTP) [reply core/cloudflow-operator/src/main/scala/cloudflow/operator/Settings.scala:135]: agree that would be nicer, I'll bind it to the operator configuration 👍

Comment (andreaTP) [reply core/cloudflow-operator/src/main/scala/cloudflow/operator/Settings.scala]: @RayRoestenburg done here:
https://github.com/lightbend/cloudflow/pull/1094/commits/c7e5386844e0dfa178ea53117c24e90884304969

Comment (RayRoestenburg) [reply core/cloudflow-operator/src/main/scala/cloudflow/operator/Settings.scala]: Thanks!

**Pull request merged**